cybered/hashing/templates/hashing/begin.html

```diff
@@ -2,9 +2,9 @@
 <html>
 {% include "includes/head.html" %}
 
-<body>
+<body class="d-flex flex-column">
```

Comment:
Adding this to *every* `<body>` feels wrong. There should be a way to redo the templates. There is [template inheritance](https://docs.djangoproject.com/en/2.1/ref/templates/language/#template-inheritance) that we might be able to use to good effect.

Reply:
I feel like it would be more of a custom class that can be used as 'cybered-page' or something? 
Could we put the <body..> tag in head.html, and then </body> in footer.html?